packages/error-reporting/system-test/testAuthClient.js

```diff
@@ -357,3 +359,69 @@ describe('Expected Behavior', function() {
     });
   });
 });
+
+describe('error-reporting', function() {
+  const TIMESTAMP = Date.now();
+  const BASE_NAME = 'error-reporting-system-test';
+  function buildName(suffix) {
+    return [TIMESTAMP, BASE_NAME, suffix].join('_');
+  }
+
+  const SERVICE_NAME = buildName('service-name');
+  const SERVICE_VERSION = buildName('service-version');
+
+  var errors;
+  var transport;
+  before(function() {
+    errors = require('../src/index.js')({
+      ignoreEnvironmentCheck: true,
+      serviceContext: {
+        service: SERVICE_NAME,
+        version: SERVICE_VERSION
+      }
+    });
+    transport = new ErrorsApiTransport(errors._config, errors._logger);
+  });
+
+  after(function(done) {
+    transport.deleteAllEvents(function(err) {
```

Comment:
By the time the call returns the events have just been registered to be deleted, but will be deleted at some point in the future.  I wrote the code this way because the api only has publish capabilities and thus verifying deletion of the events is not necessary.